styleguide/assets/variables.css

```
@@ -37,6 +37,8 @@
 	--color-box-shadow: rgba(77, 77, 77, 0.5);
 	--color-border: #ededed;
 	--color-border-dark: #dbdbdb;
+	--color-main-background-semitransparent: rgba(255,255,255,0.8);
+	--background-blur: blur(10px);
```

Let’s go with slightly higher, as per @marcoambrosini feedback :)
```suggestion
	--background-blur: blur(25px);
```

---

yes, Julius added new variables https://github.com/nextcloud/server/pull/33610/files and i will use his :)